### PR TITLE
Support pushing build-definitions images to quay.io/konflux-ci

### PR DESCRIPTION
**Pull request by chmeliik**
[STONEBLD-2339](https://issues.redhat.com//browse/STONEBLD-2339)

See individual commits for details

### CODE REVIEW

**Comment by chmeliik** [comment]
/hold

Needs to be merged not long before https://github.com/konflux-ci/build-definitions/pull/1041

File: components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pr-cronjob.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: redhat-appstudio-tekton-catalog-pr-cronjob
+  name: konflux-ci-pr-cronjob
```
Comment (mmorhun) [inline components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pr-cronjob.yaml:4]: `pr` is ambiguous. Better use pull-request or merge-request.

Comment (chmeliik) [reply components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pr-cronjob.yaml]: Renamed to `konflux-ci-pull-request-builds-pruner`

File: components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pull-request-builds-pruner-token.yaml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-  name: redhat-appstudio-tekton-catalog-pr-token
+  name: konflux-ci-pull-request-builds-pruner-token
   annotations:
     argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
     argocd.argoproj.io/sync-wave: "-1"
 spec:
   dataFrom:
     - extract:
-        key: production/build/redhat-appstudio-tekton-catalog-pr-token
+        key: production/build/konflux-ci-pr-builds-pruner-token
```
Comment (mmorhun) [inline components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pull-request-builds-pruner-token.yaml:11]: please expand `pr` here as well

**Comment by chmeliik** [comment]
Did the tests not run? :thinking: 

/test all

**Review by mmorhun — APPROVED** [review]

File: components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pull-request-builds-pruner-token.yaml (hunk 2 above)
Comment (mmorhun) [reply components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pull-request-builds-pruner-token.yaml:11]: It's external secret, so never mind

**Comment by chmeliik** [comment]
/hold

Let's try again on Monday :smiling_face_with_tear:

**Review by lkolacek — APPROVED** [review]
LGTM

**Pull request merged**